### ISSUE TITLE
Proposal to change the way we propagate PRs to different branches

### ISSUE THREAD

**Issue by andresailer** [DIRACGrid/DIRAC#4941]
Instead of merging all PRs to one branch (e.g., rel-v7r0) and then merging rel-v7r0 into rel-v7r1 etc.
I propose we use one of the couple "backport" github actions to apply merged PRs to the other branches, which can be usually chosen by labeling the PRs. And once the PR is merged the github action will create a new PR targeting the other branches.

Pros:
+ Testing individual PRs not just against the original branch, but all others they apply to.
+ We can finally do the "do not apply this to other branches" fixes
+ Merge conflicts are solved by the person creating the original PR, not all by whoever merges the branches
+ No more cases of "bad merges"

Cons:
- Slightly more work for whoever doesn't merge the branches

**Comment by atsareg** [comment]
Looks like a good proposal. This would also make developers think twice before making a PR against branches 3 stages before the integration branch :)

**Comment by andresailer** [comment]
Thanks for all the ❤️ s. 😃 I'll try to make some tests with the github actions for the next BiLD.

**Comment by fstagni** [comment]
Linking discussion in https://demo.hedgedoc.org/pJhj2yHvT7iwJQw4tvH1mg?both#PRs-merging-strategy

**Comment by andresailer** [comment]
Fixed
